website/docs/configuration/functions/coalesce.html.md

```diff
@@ -13,8 +13,8 @@ description: |-
 earlier, see
 [0.11 Configuration Language: Interpolation Syntax](../../configuration-0-11/interpolation.html).
 
-`coalesce` takes any number of string arguments and returns the first one
-that isn't empty.
+`coalesce` takes any number of arguments and returns the first one
+that isn't null or empty.
```

Comment:
I think we need to be specific here about empty _string_, because this doesn't treat empty lists or maps as if "null".